handlers/main.yml

```diff
@@ -1,36 +1,41 @@
 ---
 - name: Restart BeeGFS mgmt service
-  service:
+  systemd:
     name: beegfs-mgmtd
+    daemon-reload: true
     state: restarted
   become: true
   when: beegfs_enable.mgmt | bool
 
 - name: Restart BeeGFS meta service
-  service:
+  systemd:
     name: beegfs-meta
+    daemon-reload: true
     state: restarted
   become: true
   when: beegfs_enable.meta | bool
 
 - name: Restart BeeGFS storage service
-  service:
+  systemd:
     name: "beegfs-storage@{{ item.port }}"
+    daemon-reload: true
     state: restarted
   become: true
   with_items: "{{ beegfs_oss }}"
   when: beegfs_enable.oss | bool
 
 - name: Restart BeeGFS admon service
-  service:
+  systemd:
     name: beegfs-admon
+    daemon-reload: true
```

Comment:
I'm assuming this is introduced if the package has been updated.  But wouldn't the package installation scripts do this update?  Have you seen situations where a systemd daemon reload was required?

Reply:
Yes, I notices that the storage services were failing to start because systemctl daemon reload hadn't been invoked.